object_vhacd.py

```diff
@@ -149,7 +149,7 @@ class VHACD_OT_VHACD(bpy.types.Operator):
     bl_idname = 'object.vhacd'
     bl_label = 'Convex Hull (V-HACD)'
     bl_description = 'Create accurate convex hulls using Hierarchical Approximate Convex Decomposition'
-    bl_options = {'REGISTER', 'PRESET'} # PRESET doesn't seem to require AdPresetBase operator to work anymore...
+    bl_options = {'REGISTER', 'PRESET', 'UNDO'} # PRESET doesn't seem to require AdPresetBase operator to work anymore...
```

Comment:
Let me know if this is unwanted; I've found it useful to be able to undo generating the meshes.